daemon/events/metrics.go

```diff
@@ -0,0 +1,21 @@
+package events
+
+import (
+	"github.com/docker/docker/dockerversion"
+	"github.com/docker/go-metrics"
+)
+
+var (
+	eventsCounter    metrics.Counter
+	eventSubscribers metrics.Gauge
+)
+
+func init() {
+	ns := metrics.NewNamespace("engine", "events", metrics.Labels{
+		"version": dockerversion.Version,
```

Comment:
@beorn7 @juliusv Please provide guidance on avoiding including commit and version on every metric.

(from stevvooe)